src/objects/logo-group/logo-group.scss

```diff
@@ -0,0 +1,16 @@
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/ms';
+
+.o-logo-group {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: size.$l;
```

Comment:
This should probably be handled by the `o-container` object wrapping the logo group instead of the logo group itself?

Reply:
`o-container` has a max-width of 80em. In the prototype, the containing element for the logos has a max-width of 60em. Should I create an `o-container` modifier with `max-width: 60em`?

Reply:
The first question I'd ask is whether or not that difference is meaningful. Does the pattern not work well with one of the existing container sizes? And is that something that could be solved by adjusting the size of the logos?

If the size difference is meaningful, then we should create a modifier with a name derived from what that difference is.